src/test/java/org/libgit2/jagged/BlobTest.java

```diff
@@ -77,14 +77,7 @@ public void testGetFilteredContent()
     {
         byte[] expected;
 
-        if (Platform.getCurrentPlatform().getOperatingSystem().equals(OperatingSystem.WINDOWS))
-        {
-            expected = "This is file two!\r\n".getBytes();
-        }
-        else
-        {
-            expected = "This is file two!\n".getBytes();
-        }
+        expected = "This is file two!\n".getBytes();
```

Comment:
#32 now updates the test environment correctly, so the original test should pass in your environment.